Formula/t/texlive.rb

```diff
@@ -470,8 +478,7 @@ def install
     end
 
     # Wrap some Python scripts so they can find dependencies and fix depythontex.
-    python_path = libexec/Language::Python.site_packages(python3)
-    ENV.prepend_path "PYTHONPATH", python_path
+    ENV.prepend_path "PYTHONPATH", venv.site_packages
```

Comment:
Might've been better to rewrite their shebangs to use the venv Python. Maybe next rebuild.